persistence-api/src/main/java/org/apache/cassandra/stargate/schema/CQLType.java

```diff
@@ -1,277 +0,0 @@
-package org.apache.cassandra.stargate.schema;
```

Comment:
I think you should not remove those files because it will clash with changes it this PR:
https://github.com/stargate/stargate/pull/349/files

Reply:
Note that in the history of this PR, there's a merge with #349 branch. 

When I merge #349, the `cdc-basic-kafka` will break.

Reply:
Can we follow the existing process to not introduce conflicts?
I think that we should merge the https://github.com/stargate/stargate/pull/349 into `cdc-basic`
and then pull the `cdc-basic` into this `cdc-basic-kafka`.
Then, based on the `cdc-basic-kafka` create `cdc-basic-kafka-schema`. 
I was following this workflow for all kafka related work.

Reply:
I've merged #349, feel free to update `cdc-basic-kafka` branch and this one.